edweiss-app/components/courses/MaterialDisplay.tsx

```diff
@@ -0,0 +1,93 @@
+import ReactComponent from '@/constants/Component';
+
+import TView from '@/components/core/containers/TView';
+import TText from '@/components/core/TText';
+import { iconSizes } from '@/constants/Sizes';
+import { Material } from '@/model/school/courses';
+import TTouchableOpacity from '../core/containers/TTouchableOpacity';
+import Icon from '../core/Icon';
+
+
+// Icons
+const slidesIcon = 'albums-outline';
+const exerciseIcon = 'document-text-outline';
+const feedbackIcon = 'arrow-undo-outline';
+const otherIcon = 'attach-outline';
+
+// Tests Tags
+export const testIDs = {
+    materialTitle: 'material-title',
+    materialDescription: 'material-description',
+    slidesTouchable: 'slides-touchable',
+    slidesIcon: 'slides-icon',
+    slidesText: 'slides-text',
+    exercisesTouchable: 'exercises-touchable',
+    exercisesIcon: 'exercises-icon',
+    exercisesText: 'exercises-text',
+    otherTouchable: 'other-touchable',
+    otherIcon: 'others-icon',
+    otherText: 'other-text',
+    feedbacksTouchable: 'feedbacks-touchable',
+    feedbacksIcon: 'feedbacks-icon',
+    feedbacksText: 'feedbacks-text',
+};
+
+/**
+ * MaterialDisplay Component
+ * 
+ * This component is responsible for displaying a material in the course page.
+ * 
+ * @param item - The material data to be displayed.
+ * @param key - The key of the assignment in the list.
+ * 
+ * @returns JSX.Element - The rendered component for the assignment display.
+ */
+const MaterialDisplay: ReactComponent<{ item: Material; }> = ({ item }) => {
+
+    const formatDateRange = (fromSeconds: number, toSeconds: number) => {
```

Comment:
Hello! Yes there is a reason why formatterOptions doesn't define `year: 'numeric'` it is because I didn't wanted to repeat 2 times the same year e.g. display `Sept 9 - Sept 15 2024` instead of `Sept 9 2024 - Sept 15 2024`

Reply:
Does specifying the year really necessary? I can hardly find a use for it. It just seems to take more space

Reply:
Yeah I agree I will do that!